src/constants/constants.ts

```diff
@@ -38,3 +38,7 @@ export const STATUSES = [
     'completed',
     'done',
 ];
+export const MSG_ON_100_PROGRESS =
+    'Proceeding further will make task progress 100%.';
+export const MSG_ON_0_PROGRESS =
+    'Proceeding further will make task progress 0%.';
```

Comment:
A nitpick:
Maybe you can put both the constants you defined under a single object?
`const PROGRESS_MSG = {
         "0": "msg1",
         "100": "msg2" 
 }
`